### DISCUSSION TITLE
It would be nice if JSVM $template wouldn't cache the files

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#3969] [Q&A]
When developing on a Raspberry Pi I'm finding that it would be better if the templates were loaded and parsed on every request instead of restarting the process.  So to do that I simply would place the views folder somewhere other than in `__hooks`, but then these are still cached.

I like the template registry though.  I think that simply being able to create a new one to throw away would solve it.  I would just put it in my template helper.

```golang
module.exports = (...frags) => {
  const tplDir = `${__hooks}/views`
  const paths = frags.map(frag => `${tplDir}/${frag}.html`)
  return $template.loadFiles(...paths)
}
```

**Comment by VictorioBerra** [comment]
I am feeling the pain of this too. This is making development horribly slow for me with AlpineJS.

Ideally:

```js
    const html = $template
      .loadFiles(`${__hooks}/views/layout.html`, `${__hooks}/views/login.html`)
      .cache(false) // NEW!!!!
      .render({
        authRecord: record,
        accountCreated,
      });

    return c.html(200, html);
```
